test/resources/statefulset.go

```diff
@@ -184,10 +193,48 @@ func (t *TestStatefulset) DeletePodAndWait() {
 	time.Sleep(60 * time.Second)
 }
 
-func (t *TestStatefulset) Cleanup() {
+func (t *TestStatefulset) Cleanup(timeout int) {
```

Comment:
Use `time.Duration` for clarity. At the call point, this looks like a magic number and it's purpose is unclear.

Reply:
Even better to use a `context.Context` to manage the timeout.

Reply:
I went with `time.Duration` because the method was relying on external and internal functions that were accepting duration for pooling.